backend-api/app/routes/consumers.py

```diff
@@ -6,24 +6,32 @@
 
 router = APIRouter()
 
-consumer_dict = {"ethereum_address": "somewhareOnTheBlockchain"}
+consumer_dict = {"eth_address": "somewhareOnTheBlockchain"}
 
 
 @router.post("/", status_code=status.HTTP_201_CREATED)
 async def create_consumer(consumer: ConsumerCreate):
+    # Create database instance, retrieve ID
+
     return ConsumerRead(id=random.randint(1, 1000), **consumer_dict)
 
 
-@router.get("/me")
-async def read_consumer():
+@router.get("/{eth_address}}", status_code=status.HTTP_200_OK)
+async def read_consumer(eth_address: str):
+    # Query database instance
+
     return ConsumerRead(id=random.randint(1, 1000), **consumer_dict)
 
 
-@router.patch("/me")
-async def update_consumer(consumer: ConsumerUpdate):
-    return ConsumerRead(id=random.randint(1, 1000), **consumer_dict)
+@router.patch("/{eth_address}", status_code=status.HTTP_204_NO_CONTENT)
```

Comment:
I didn't put 204 NO CONTENT cuz I usually like returning the updated object after a post/put/patch request since it's to verify that the update was successful. I use 204 when deleting objects since there's no object to look at anymore.